pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -810,7 +810,7 @@ func newOauthClient(tokenSource oauth2.TokenSource) (*http.Client, error) {
 		glog.Infof("Using existing Token Source %#v", tokenSource)
 	}
 
-	if err := wait.PollImmediate(5*time.Second, 30*time.Second, func() (bool, error) {
+	if err := wait.PollImmediate(5*time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
Have we asked anyone why we should need this many retries? It seems like this should very rarely fail on the first try.

Reply:
yeah, I'm also a bit skeptical that something that fails after 30s of retries will pass with 5m of retries.